benchmark/README.md

```diff
@@ -0,0 +1,5 @@
+# restify-benchmark
+
+```sh
+npm start
```

Comment:
Can this be a make task instead? I'd rather not split our build tasks between npm tasks and makefiles.

Reply:
Yes, the current benchmark is living in a separate folder: it has it's own npm package file as well. I can move these dependencies to the main devDependencies and create a make command.

What do you think? Should I remove the separate `package.json`?

Reply:
Yeah probably makes sense to make those a devDependency in the main `package.json`

Reply:
Done, I merged it into the main package.json, also added make command and mentioned in the `CONTRIBUTION.md`